logic/account.js

```diff
@@ -404,9 +404,10 @@ class Account {
 		const totalSupplyBignum = new Bignum(totalSupply);
 		const approvalBignum = votersBalanceBignum
 			.dividedBy(totalSupplyBignum)
-			.times(100)
-			.round(2);
-		return !approvalBignum.isNaN() ? approvalBignum.toNumber() : 0;
+			.multipliedBy(100)
+			.toFixed(2);
```

Comment:
`.decimalPlaces(2)` instead of `.toFixed(2)` is better here because is not converting to `string`